components/link/stories/link.stories.js

```diff
@@ -54,8 +54,25 @@ export default {
 			},
 			control: "boolean",
 		},
+		isInline: {
+			name: "Inline",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+		},
 		staticColor,
-		isQuiet,
+		isQuiet: {
+			name: "Quiet styling",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			if: { arg: "isInline", neq: true },
+		},
```

Comment:
```suggestion
		isQuiet: {
			...isQuiet,
			if: { arg: "isInline", neq: true },
		},
```

Just a note, you can keep it aligned with the shared isQuiet variants by spreading the original object and adding your if statement.